README.md

```diff
@@ -86,7 +86,7 @@ For more information about each individual setting checkout Release::Notes's
 ### TL;DR
 
 ```sh
-bundle exec update_release_notes:run
+rails update_release_notes:run
```

Comment:
If used outside of rails this would not be valid. Perhaps just updating the docs as such:

```suggestion
###TL;DR

rails
``sh
bin/rails update_release_notes:run
``
non-rails
``sh
bundle exec rake update_release_notes:run
``
```